pkgs/development/libraries/quarto/default.nix

```diff
@@ -0,0 +1,56 @@
+{ callPackage
+, lib
+, pandoc
+, esbuild
+, deno
+, stdenv
+, fetchurl
+, nixos
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "quarto";
+  version = "1.0.38";
+
+  src = fetchurl {
+    url = "https://github.com/quarto-dev/quarto-cli/releases/download/v1.0.38/quarto-1.0.38-linux-amd64.tar.gz";
+    sha256 = "sha256-bqEGRX6ihyrgmIxqkV9jr/2QfWM5LWnsjoGpTqK9qOc=";
+  };
+
+  buildInputs = [
+      pandoc
+      deno
+      esbuild
+  ];
+
+  installPhase = ''
+      mkdir src
+      tar -zxvf ${finalAttrs.src} -C src
+      mkdir -p $out/bin
+      mkdir -p $out/share
+
+      rm src/quarto-${finalAttrs.version}/bin/tools/deno
+      rm src/quarto-${finalAttrs.version}/bin/tools/pandoc
+      rm src/quarto-${finalAttrs.version}/bin/tools/esbuild
+
+      mv src/quarto-${finalAttrs.version}/bin/* $out/bin
+      mv src/quarto-${finalAttrs.version}/share/* $out/share
```

Comment:
```suggestion
      rm src/quarto-*/bin/tools/deno
      rm src/quarto-*/bin/tools/pandoc
      rm src/quarto-*/bin/tools/esbuild

      mv src/quarto-*/bin/* $out/bin
      mv src/quarto-*/share/* $out/share
```

Reply:
Why do you want this change? Both are equivalent solutions? Do I miss something?

Reply:
Anyway, during the missed unpackPhase this section is not relevant anymore.

Reply:
Solved in [152051c1949b0717f59d072330ecc83bccd1fe6e](https://github.com/NixOS/nixpkgs/pull/186697/commits/152051c1949b0717f59d072330ecc83bccd1fe6e)